### PR TITLE
Update eslint config and apply it

### PR DESCRIPTION
**Pull request by AlasDiablo**

### CODE REVIEW

File: tdm-be/src/controller/traitment.ts [hunk 1]
```diff
@@ -17,11 +11,19 @@ import {
     HTTP_PRECONDITION_REQUIRED,
 } from '~/lib/http';
 import logger from '~/lib/logger';
+import type { Processing } from '~/model/ProcessingModel';
 import { createProcessing, findProcessing, updateProcessing } from '~/model/ProcessingModel';
```
Comment (parmentf) [inline tdm-be/src/controller/traitment.ts:15]: **question:** ‏I thought types were more isolated

Usually, there was a blank line at least before the `import type`...

File: tdm-be/src/worker/enrichment.ts [hunk 2]
```diff
@@ -15,6 +12,12 @@ import { findProcessing } from '~/model/ProcessingModel';
 import { updateProcessing } from '~/model/ProcessingModel';
 import Status from '~/model/Status';
 
+import axios from 'axios';
+
+import { readFile } from 'node:fs/promises';
```
Comment (parmentf) [inline tdm-be/src/worker/enrichment.ts:17]: **question:** ‏Why so many blank lines?

File: tdm-be/src/worker/fields/csvFields.ts [hunk 3]
```diff
@@ -1,6 +1,8 @@
+import logger from '~/lib/logger';
+
 import axios from 'axios';
+
 import { readFile } from 'node:fs/promises';
```
Comment (parmentf) [inline tdm-be/src/worker/fields/csvFields.ts:5]: **question:** Are external packages and node packages separated?‏

**Review by parmentf — APPROVED** [review]
Few questions...

File: tdm-be/src/controller/traitment.ts (hunk 1 above)
Comment (AlasDiablo) [reply tdm-be/src/controller/traitment.ts:15]: import/order miss understand the import with ts-paths

File: tdm-be/src/worker/enrichment.ts (hunk 2 above)
Comment (AlasDiablo) [reply tdm-be/src/worker/enrichment.ts:17]: i have add a new line for each new type of import

File: tdm-be/src/worker/fields/csvFields.ts (hunk 3 above)
Comment (AlasDiablo) [reply tdm-be/src/worker/fields/csvFields.ts:5]: external and buildins is considered as two different groups :/

**Pull request merged**